app/src/main/java/ar/com/wolox/android/example/ui/example/presenter/NewsPresenter.kt

```diff
@@ -0,0 +1,33 @@
+package ar.com.wolox.android.example.ui.example.presenter
+
+import ar.com.wolox.android.databinding.ItemNewBinding
+import ar.com.wolox.android.example.model.News
+import ar.com.wolox.android.example.ui.example.view.NewsView
+import ar.com.wolox.android.example.ui.example.viewholder.NewsViewHolder
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import javax.inject.Inject
+
+class NewsPresenter @Inject constructor() : BasePresenter<NewsView>(){
+
+    private var items : MutableList<News> = mutableListOf()
+    private lateinit var bindingHolder: ItemNewBinding
+
+    //Agregar el binding
+    fun addBinding(binding : ItemNewBinding){
+        bindingHolder=binding
+    }
```

Comment:
The presenter should never have access to the view. You should create an interface in your item and implement it in the presenter as a listener